jsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+    "compilerOptions": {
+      "paths": {
+        "@/*": [
```

Comment:
i dont think you need this file. you already have tsconfig

Reply:
deleted